filament/backend/src/BackendUtils.cpp

```diff
@@ -591,4 +591,47 @@ CString to_string<filament::backend::TargetBufferFlags>(filament::backend::Targe
     return { string, 6 };
 }
 
+
+template<>
+CString to_string<filament::backend::TextureFormat>(filament::backend::TextureFormat value) noexcept {
```

Comment:
could you return a std::string_view<> instead? All these strings are static strings, so no need to do a heap allocation for them.

also I'm a little confused because this is a cpp file, so where is `to_string` declared?